apps/aecore/src/aec_peer_messages.erl

```diff
@@ -199,7 +223,40 @@ latest_vsn(get_node_info)        -> ?GET_NODE_INFO_VSN;
 latest_vsn(node_info)            -> ?NODE_INFO_VSN;
 latest_vsn(close)                -> ?CLOSE_VSN.
 
-deserialize(ping, Vsn, PingFlds) when Vsn == ?PING_VSN ->
+%% In cases where we support multiple message versions.
+%% Only add clauses for messages where multiple vsns are needed.
+%% Default is to support only the latest version.
+msg_versions(ping) ->
+    {ok, Min} = aeu_env:find_config([<<"sync">>, <<"minimum_ping_version">>],
+				    [user_config, schema_default]),
+    [V || V <- [?PING_VSN_1, ?PING_VSN_2],
+	  V >= Min];
+msg_versions(_Msg) ->
+    undefined.
+
+all_msg_versions() ->
```

Comment:
Not sure I understand the purpose and/or the name of this function, it is not all messages?

Reply:
I'll revisit. May well be remnants of me trying to make up my mind on how to structure things.

Reply:
I've renamed it to supported_msg_versions is that clearer?